### PR TITLE
Fix expired tls certs for cpp tests

### PR DESCRIPTION
**Pull request by sijie**
*Problem*

The current master is broken due to an expired CA cert is used in CPP tests.

*Modification*

Use the certs we used for integration tests to keep it consistent across the rep.

### CODE REVIEW

File: pulsar-broker/src/test/resources/authentication/tls/cacert.pem [hunk 1]
```diff
@@ -1,62 +1,29 @@
-Certificate:
-    Data:
-        Version: 3 (0x2)
-        Serial Number:
-            88:08:98:b3:13:d8:00:94
-        Signature Algorithm: sha1WithRSAEncryption
-        Issuer: C=US, ST=CA, O=Apache, OU=Pulsar Incubator, CN=localhost
-        Validity
-            Not Before: Feb 17 01:37:33 2018 GMT
-            Not After : Feb 16 01:37:33 2021 GMT
```
Comment (BewareMyPower) [inline pulsar-broker/src/test/resources/authentication/tls/cacert.pem:10]: I'm not familiar with TLS certificate but just have a question. The original `Certificate - Data - Validity` block has the `Not After` field that might lead to the broken CI. After this change, should we also keep the header-like  block to record the `Not After` time?

I think the root cause is #1244 that intended to create certs which will expire after year 2030. However, the `cacert.pem` still expired after 2021-02-16.

**Comment by lhotari** [comment]
btw. Some of the previous test keys/certificates weren't TLS 1.3 compatible. I assume that this problem gets also fixed? 
TLS 1.3 dropped support for SHA-1 signature hashes and DSA keys and will reject those. The rejection error message might be about invalid certificate.

**Review by jiazhai — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Review by tuteng — APPROVED** [review]

**Review by eolivelli — APPROVED** [review]
looks good

**Review by merlimat — APPROVED** [review]

**Comment by merlimat** [comment]
@sijie There are still test failing because clients are validating the "hostname" with the CN of the certificate. The tests expect the certificates to be issued with CN `localhost`.

**Comment by merlimat** [comment]
I've regenerated the certs with "localhost" for broker CN

**Comment by sijie** [comment]
@merlimat thank you!

**Review by jerrypeng — APPROVED** [review]

**Pull request merged**